backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -49,6 +49,8 @@ public UsersController(WebClientService webClientService) {
     @GetMapping("/users/{id}")
     public Mono<Object> users(@PathVariable String id) {
         String path = usersPath + "/" + id;
-        return webClientService.get(path, null);
+        Mono<Object> mono = webClientService.get(path, null);
+        Object data = mono.block();
+        return mono;
```

Comment:
This change isn't necessary and wouldn't be included in the merged pull request. I just wanted to show that we can block now.